fizzbuzz.js

```diff
@@ -0,0 +1,7 @@
+const fizzbuzz = (max) => {
+  for (let i = 1; i <= max; i++) {
+    (i % 3 == 0 && i % 5 == 0) ? console.log('FizzBuzz') : (i % 3 == 0) ? console.log('Fizz') : (i % 5 == 0) ? console.log('Buzz') : console.log(i)
```

Comment:
If you're aiming for brevity, consider moving the ternary operations inside a single `console.log` statement.

```js
console.log( (i % 3 == 0 && i % 5 == 0) ? "FizzBuzz" : ... )
```